packages/dds/tree/src/test/tableSchema.spec.ts

```diff
@@ -91,7 +91,7 @@ describe("TableFactory unit tests", () => {
 
 			// TODO: ideally the "props" property would not exist at all on the derived class.
 			// For now, it is at least an optional property and cannot be set to anything meaningful.
-			type _test = requireAssignableTo<null | undefined, Column["props"]>;
+			type _test = requireAssignableTo<undefined, Column["props"]>;
```

Comment:
`requireTrue<areSaflyAssignable<` here would check both that Column["props"] allows undefined and that its assignable to undefined, and that its not any. I think that would be a better check for this case and the one below

Reply:
as is this only checks that Column["props"] allows undefined,. THis would apss if its type was `any`, `Foo | undefined` etc.